src/server/scripts/Kalimdor/zone_silithus.cpp

```diff
@@ -1214,9 +1217,29 @@ class go_wind_stone : public GameObjectScript
 
                 void SummonNPC(GameObject* go, Player* player, uint32 npc, uint32 spell)
                 {
-                    player->CastSpell(player, spell);
+                    isSummoning = true;
+                    player->CastSpell(player, spell, true);
                     TempSummon* summons = go->SummonCreature(npc, go->GetPositionX(), go->GetPositionY(), go->GetPositionZ(), player->GetOrientation() - float(M_PI), TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 10min);
                     summons->CastSpell(summons, SPELL_SPAWN_IN, false);
+
+                    // Add player to SmartAI stored target list 1
+                    ObjectVector primaryTarget;
+                    primaryTarget.push_back(player);
+                    ENSURE_AI(SmartAI, summons->AI())->GetScript()->AddToStoredTargetList(primaryTarget, 1);
+
+                    // Add player's group (if applicable) within 50yds to SmartAI stored target list 2
+                    if (Group* group = player->GetGroup())
+                    {
+                        ObjectVector secondaryTargets;
+                        for (GroupReference* groupRef = group->GetFirstMember(); groupRef != nullptr; groupRef = groupRef->next())
+                            if (Player* member = groupRef->GetSource())
+                                if (member != player && member->IsInMap(summons) and member->GetDistance(summons) < 50.0f)
```

Comment:
We don't use named logical operators, please replace `and` with `&&`

Reply:
That was an oopsie on my part, fixed. I guess there is a bug in Visual Studio since it shouldn't allow to do that when permissive-flag is explicitly disabled.